x-pack/solutions/observability/plugins/observability/server/lib/esql_extensions/set_esql_recommended_queries.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+import { PluginSetup as ESQLSetup } from '@kbn/esql/server';
+
+const TRACES_ESQL_RECOMMENDED_QUERIES = [
+  {
+    name: i18n.translate('xpack.observability.recommendedQueries.findRootTraces.name', {
+      defaultMessage: 'Root traces',
+    }),
+    query: 'FROM traces-* | WHERE parent.id IS NULL',
+    description: i18n.translate(
+      'xpack.observability.recommendedQueries.findRootTraces.description',
+      {
+        defaultMessage:
+          'Display all root spans, which represent the start of a request or transaction. This is useful for getting a high-level overview of all operations in your system.',
```

Comment:
Should we remove `transaction` here?

```suggestion
          'Display all root spans, which represent the start of a request. This is useful for getting a high-level overview of all operations in your system.',
```